extensions/vault/runtime/src/main/java/io/quarkus/vault/internal/LeaseBase.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vault.internal;
+
+import static io.quarkus.vault.internal.LogConfidentialityLevel.MEDIUM;
+
+public class LeaseBase extends TimeLimitedBase {
+
+    public String lease_id;
```

Comment:
`leaseId`.

Reply:
done. I reviewed all model objects and got rid of the '_'.
initially it was to ease json databinding. I added JsonbProperty annotations where needed.